packages/blockchain-wallet-v4-frontend/src/components/OrderStatus/index.js

```diff
@@ -147,6 +156,13 @@ export const OrderNote = ({ status }) => {
           defaultMessage='We have refunded your wallet after the trade failed.  Please return to the swap tab to start a new trade.'
         />
       )
+    case DELAYED:
+      return (
+        <FormattedMessage
+          id='components.orderstatus.note.swapdelayed'
+          defaultMessage='Don’t worry, your exchange is in process. Swap trades are completed on-chain. If transaction volumes are high, there are sometimes delays.'
```

Comment:
Do delayed trades eventually arrive?

in process -> in progress?

Reply:
I think the assumption is that they do. I pulled the copy off [here](https://www.figma.com/file/efs6yYGbwy6Lbr48IxTAYUpP/Sprint-Art?node-id=3%3A574), but in progress makes more sense.